### DISCUSSION TITLE
new version release requirements

### DISCUSSION THREAD

**Discussion by fgardt** [containrrr/watchtower#836] [Q&A] [answered]
What are the requirements to publish a new release of watchtower?
I'm waiting for a release that includes #757.

Alternatively, what is the `latest-dev` image tag updated to?
I'm missing any documentation about the image tags

**Comment by simonaronsson** [comment, marked as answer]
Hi!

Thank you for the question!

- `latest-dev` is continuously built from `main` and as such, may or may not be stable 😂 
- `latest` is updated with every new release, although this does not happen that often due to the limited amount of spare time we're able to sink into the project.

Best,
Simme
